src/Microsoft.SqlTools.Serialization/SerializationService.cs

```diff
@@ -0,0 +1,96 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.SqlTools.Hosting.Contracts;
+using Microsoft.SqlTools.Hosting.Protocol;
+using Microsoft.SqlTools.Utility;
+
+namespace Microsoft.SqlTools.Serialization
+{
+    /// <summary>
+    /// Service responsible for securing credentials in a platform-neutral manner. This provides
+    /// a generic API for read, save and delete credentials
+    /// </summary>
+    public class SerializationService
```

Comment:
For newer services, please consider using the HostedService pattern and avoid singleton instance definitions. See `TSqlFormatterService` for an example. You can just change it to 

```
[Export(typeof(IHostedService))]
public class SerializationService: HostedService<SerializationService>, IComposableService
```